front-end/src/app/app.component.html

```diff
@@ -1,9 +1,9 @@
 <main class="d-flex flex-nowrap">
     <app-sidebar class="d-flex flex-column flex-shrink-0 p-3 text-bg-dark col-sm-3" style="width: 280px;"></app-sidebar>
-    <div class="col-sm-9">
+    <div class="col-sm-8">
```

Comment:
Why was this switched to 8 from 9? Curious what the reasoning was.

Reply:
mistake